Dockerfile

```diff
@@ -0,0 +1,100 @@
+FROM ubuntu:14.04
+MAINTAINER Bilal Salam <bilal.salam@kitware.com>
+
+
+RUN apt-get update && \
+apt-get install -y \
+git \
+wget \
+python-qt4 \
+openslide-tools python-openslide \
+build-essential \
+swig \
+make \
+zlib1g-dev \
+curl \
+libcurl4-openssl-dev \
+libexpat1-dev \
+unzip \
+libhdf5-dev \
+libjpeg-dev \
+libopenslide-dev \
+libpng12-dev \
+libpython3-dev \
+libtiff5-dev \
+cmake \
+openslide-tools && \
+apt-get autoremove && \
+rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+WORKDIR /
```

Comment:
Need to check if the WORKDIR command make Docker create a cached layer. If so, we can just put a `cd` at the beginning of the RUN command to move to a directory